components/layout/construct.rs

```diff
@@ -54,6 +54,7 @@ use style::properties::{self, ServoComputedValues};
 use style::selector_parser::{PseudoElement, RestyleDamage};
 use style::servo::restyle_damage::{BUBBLE_ISIZES, RECONSTRUCT_FLOW};
 use style::stylist::Stylist;
+use style::values::{Either, None_};
```

Comment:
I don't think the type would be necessary. It's not gonna be used anywhere anyway. Instead, you can just bind it to a name, like so

``` rust
Either::Second(_none) => (),
```